pydicom/tests/test_rle_pixel_data.py

```diff
@@ -1,126 +1,762 @@
+EMRI_EXPL_LITTLE_10F = get_testdata_files("emri_small.dcm")[0]
+EMRI_RLE_10F = get_testdata_files("emri_small_RLE.dcm")[0]
+# 16/16-bit, 3 sample/pixel, 1 frame
+SC_EXPL_LITTLE_16_1F = get_testdata_files("SC_rgb_16bit.dcm")[0]
+SC_RLE_16_1F = get_testdata_files("SC_rgb_rle_16bit.dcm")[0]
+# 16/16-bit, 3 sample/pixel, 2 frame
+SC_EXPL_LITTLE_16_2F = get_testdata_files("SC_rgb_16bit_2frame.dcm")[0]
+SC_RLE_16_2F = get_testdata_files("SC_rgb_rle_16bit_2frame.dcm")[0]
+# 32/32-bit, 1 sample/pixel, 1 frame
+RTDOSE_EXPL_LITTLE_1F = get_testdata_files("rtdose_1frame.dcm")[0]
+RTDOSE_RLE_1F = get_testdata_files("rtdose_rle_1frame.dcm")[0]
+# 32/32-bit, 1 sample/pixel, 15 frame
+RTDOSE_EXPL_LITTLE_15F = get_testdata_files("rtdose.dcm")[0]
+RTDOSE_RLE_15F = get_testdata_files("rtdose_rle.dcm")[0]
+# 32/32-bit, 3 sample/pixel, 1 frame
+SC_EXPL_LITTLE_32_1F = get_testdata_files("SC_rgb_32bit.dcm")[0]
+SC_RLE_32_1F = get_testdata_files("SC_rgb_rle_32bit.dcm")[0]
+# 32/32-bit, 3 sample/pixel, 2 frame
+SC_EXPL_LITTLE_32_2F = get_testdata_files("SC_rgb_32bit_2frame.dcm")[0]
+SC_RLE_32_2F = get_testdata_files("SC_rgb_rle_32bit_2frame.dcm")[0]
+
+# Transfer Syntaxes (non-retired + Explicit VR Big Endian)
+SUPPORTED_SYNTAXES = [RLELossless]
+UNSUPPORTED_SYNTAXES = [
+    '1.2.840.10008.1.2',  # Implicit VR Little Endian
+    '1.2.840.10008.1.2.1',  # Explicit VR Little Endian
+    '1.2.840.10008.1.2.1.99',  # Deflated Explicit VR Little Endian
+    '1.2.840.10008.1.2.2',  # Explicit VR Big Endian
+    '1.2.840.10008.1.2.4.50',  # JPEG Baseline (Process 1)
+    '1.2.840.10008.1.2.4.51',  # JPEG Extended (Process 2 and 4)
+    '1.2.840.10008.1.2.4.57',  # JPEG Lossless (Process 14)
+    '1.2.840.10008.1.2.4.70',  # JPEG Lossless (Process 14, Selection Value 1)
+    '1.2.840.10008.1.2.4.80',  # JPEG-LS Lossless
+    '1.2.840.10008.1.2.4.81',  # JPEG-LS Lossy (Near-Lossless)
+    '1.2.840.10008.1.2.4.90',  # JPEG 2000 Image Compression (Lossless Only)
+    '1.2.840.10008.1.2.4.91',  # JPEG 2000 Image Compression
+    '1.2.840.10008.1.2.4.92',  # JPEG 2000 Part 2 Multi-component
+    '1.2.840.10008.1.2.4.93',  # JPEG 2000 Part 2 Multi-component
+    '1.2.840.10008.1.2.4.94',  # JPIP Referenced
+    '1.2.840.10008.1.2.4.95',  # JPIP Referenced Deflate
+    '1.2.840.10008.1.2.4.100',  # MPEG2 Main Profile / Main Level
+    '1.2.840.10008.1.2.4.101',  # MPEG2 Main Profile / High Level
+    '1.2.840.10008.1.2.4.102',  # MPEG-4 AVC/H.264 High Profile / Level 4.1
+    '1.2.840.10008.1.2.4.103',  # MPEG-4 AVC/H.264 BD-compatible High Profile
+    '1.2.840.10008.1.2.4.104',  # MPEG-4 AVC/H.264 High Profile For 2D Video
+    '1.2.840.10008.1.2.4.105',  # MPEG-4 AVC/H.264 High Profile For 3D Video
+    '1.2.840.10008.1.2.4.106',  # MPEG-4 AVC/H.264 Stereo High Profile
+    '1.2.840.10008.1.2.4.107',  # HEVC/H.265 Main Profile / Level 5.1
+    '1.2.840.10008.1.2.4.108',  # HEVC/H.265 Main 10 Profile / Level 5.1
+]
+ALL_SYNTAXES = SUPPORTED_SYNTAXES + UNSUPPORTED_SYNTAXES
+
```

Comment:
Does it make sense to define ALL_SYNTAXES somewhere common, then just create UNSUPPORTED_SYNTAXES from set logic (ALL_SYNTAXES ^ SUPPORTED_SYNTAXES)?  Seems like that would avoid repetition across test files.

Reply:
Done. I'll change the numpy unit tests in another PR